### DISCUSSION TITLE
What's with Github Plans and Support?

### DISCUSSION THREAD

**Discussion by codehamster69** [community/community#140478] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Throughout the documentation of GitHub, there's a mention of "Github Pro", but there is no such plan in existence. The pricing page only shows Free, Team and Enterprise. That's it.I was going to get Github Pro, but had to choose Github Team just because of the fact.

This not ends here. I created a Support Ticket as the Team / Organization for a problem my collaborators had been facing. It's been an hour, and still no response. Is that the support you get here even after paying for it? On the pricing page, as I see, there is no option for standard support in free plan, but it exists on Teams.

![image](https://github.com/user-attachments/assets/5c33f865-3e89-451b-9374-bb47819e49c4)

What is Standard Support? If standard support is all about creating tickets and waiting times for responses, then I would like you to know that Standard Support was already there on the Free Plan. So, why is there no mention about it on the pricing page?

I think there should be some ways to reach out to Github, at least if YOU PAY.

**Comment by Topromm** [comment]
GitHub Pro used to be an individual plan, but it seems they've now streamlined the options? They should probably update the documentation to reflect on the current plans.

GitHub Team and higher plans are supposed to offer "Standard Support," which from what I understand, typically means access to ticket-based support with faster response times than the Free plan. But even with Standard Support, response times can wildly vary based on the volume of tickets, which would explain the delay.

I would try following up on the ticket if it’s been a while, but waiting a few hours to several days is sadly getting increasingly more common.

**Reply by codehamster69** [reply]
I strongly agree. They should update their docs, and they should properly differentiate the features among the plans.

What would have really been better, is a "Position in the Queue" feature, so as to know for how much time can be expected for the reply. Its been a day for me now. Let's see.

**Comment by ebndev** [comment]
Hi @codehamster69, thanks for being a part of the GitHub Community!

GitHub Pro is an option for personal accounts. It looks like you may have been upgrading for an organization. If you wanted upgrade your personal account, you can view that option [here](https://github.com/account/choose?action=upgrade).

<img width="1053" alt="Screenshot 2024-10-04 at 2 28 01 PM" src="https://github.com/user-attachments/assets/bc079fd3-f3bf-4612-aa8f-d023ea968aa6">
<img width="1336" alt="Screenshot 2024-10-04 at 2 23 46 PM" src="https://github.com/user-attachments/assets/059e6762-02cd-42d2-9182-fa9d87512961">

This doc about [GitHub’s plans](https://docs.github.com/en/get-started/learning-about-github/githubs-plans) has the most relevant information and lists the different features among plans. If there is anything else that needs to be updated, please consider [submitting a pull request](https://github.com/github/docs/blob/main/content/get-started/learning-about-github/githubs-plans.md) to help us improve our docs. Thank you!

**Reply by codehamster69** [reply]
Oh, thanks. The hyperlink you provided, helps me choose an account to go with, and I can see a "Pro" plan out there. Seems like there is no way you can reach the page through the settings. One must have the link (as it seems). Thanks, anyways.

Still, I am disappointed by the fact there is no distinctive support option when compared with the free plan and, there are no replies yet to my support ticket. My collaborators have a free plan, and they have an option for a support ticket which remains unanswered till date, even though the ticket was opened months ago. And I am about to experience the same.

**Comment by AbdulAhad365** [comment]
for your support
